static/app/views/organizationCreate/index.tsx

```diff
@@ -9,9 +12,32 @@ import ConfigStore from 'sentry/stores/configStore';
 import {OrganizationSummary} from 'sentry/types';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 
+enum RegionDisplayName {
+  US = '🇺🇸 United States of America (USA)',
```

Comment:
```suggestion
  US = '🇺🇸 United States of America (US)',
```

nit: can we use a 2-letter abbreviation here as well to match `"EU"`

Reply:
Yep, can do. I was mainly going off the figma mocks but I can change this.